contrib/hadoop-store-builder/src/java/voldemort/store/readonly/swapper/HdfsFailedFetchLock.java

```diff
@@ -108,7 +110,12 @@
     public HdfsFailedFetchLock(VoldemortConfig config, Props props) throws Exception {
         super(config, props);
         fileSystem = HadoopUtils.getHadoopFileSystem(config, clusterDir);
-        initDirs();
+        try {
+            initDirs();
+        } catch(Exception ex) {
```

Comment:
I don't know the difference between both of them. I thought Exception is basic for all java exceptions.